Add more async support

jettify opened the pull request:
PR for #92 #91

jettify commented:
Fixed issue with possible double commit. Please review.

asvetlov commented on aiopg/sa/result.py:
```diff
@@ -5,9 +5,18 @@
 
 
+try:
+    StopAsyncIteration
```
Why do you need the check?

`if PY_35:` statement above prevents `StopAsyncIteration` usage for python < 3.5

asvetlov commented:
Looks good but probably we should get rid of `StopAsyncIteration` in `utils.py` too.

jettify commented:
I have dropped  `StopAsyncIteration` but pyflakes complains with:
```python
aiopg/cursor.py:392:23: F821 undefined name 'StopAsyncIteration'
aiopg/sa/result.py:337:23: F821 undefined name 'StopAsyncIteration'
```
so I have added `# noqa` marker to those lines.

asvetlov commented:
LGTM
